tensorboard/webapp/reducer_config.ts

```diff
@@ -36,8 +36,7 @@ export function logger(reducer: ActionReducer<any>): ActionReducer<any> {
   };
 }
 
-export const metaReducers: MetaReducer<any>[] =
-  config.env === 'dev' ? [logger] : [];
```

Comment:
Seems unrelated to this PR, is making the logger enabled in prod a deliberate change?

Reply:
Oops! Reverted!